packages/edit-site/src/utils/get-is-list-page.js

```diff
@@ -14,9 +14,7 @@ export default function getIsListPage(
 	isMobileViewport
 ) {
 	return (
-		[ '/wp_template', '/wp_template_part/all', '/pages' ].includes(
-			path
-		) ||
+		[ '/wp_template', '/pages' ].includes( path ) ||
```

Comment:
Concerns about deleting this were discussed in [this comment](https://github.com/WordPress/gutenberg/pull/61080#discussion_r1580551545), but it should be possible to delete it with this PR.